Description: Fix bad ALPN check in mcrouter
 Needed for OpenSSL 1.0 on Debian 8
Author: Fred Emmott <fe@fb.com>
Origin: vendor
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: hhvm-nightly-2019.08.23/patches/mcrouter-alpn-check.patch
===================================================================
--- hhvm-nightly-2019.08.23.orig/patches/mcrouter-alpn-check.patch
+++ hhvm-nightly-2019.08.23/patches/mcrouter-alpn-check.patch
@@ -14,7 +14,7 @@ index 3930c6da..e8234cc7 100644
  #include "mcrouter/routes/ShardSplitRoute.h"
  #include "mcrouter/routes/ShardSplitter.h"

-+#ifdef FOLLY_OPENSSL_HAS_ALPN
++#if FOLLY_OPENSSL_HAS_ALPN
 +#include "mcrouter/lib/network/ThriftTransport.h"
 +#endif
 +
@@ -25,7 +25,7 @@ index 3930c6da..e8234cc7 100644
        it->second.push_back(ap);
        folly::StringPiece nameSp = it->first;

-+#ifdef FOLLY_OPENSSL_HAS_ALPN
++#if FOLLY_OPENSSL_HAS_ALPN
        if (ap->getProtocol() == mc_thrift_protocol) {
          using Transport = ThriftTransport<RouterInfo>;
          destinations.push_back(createDestinationRoute<Transport>(